src/frontend/screens/Settings/components/Shortcuts.tsx

```diff
@@ -68,6 +71,17 @@ const Shortcuts = () => {
         handleChange={() => setAddSteamShortcuts(!addSteamShortcuts)}
         title={t('setting.addgamestosteam', 'Add games to Steam automatically')}
       />
+      <ToggleSwitch
+        htmlId="shortcutsToSteamUninstalled"
+        value={addSteamShortcutsUninstalled}
+        handleChange={() =>
+          setAddSteamShortcutsUninstalled(!addSteamShortcutsUninstalled)
+        }
+        title={t(
+          'setting.addgamestosteamuninstalled',
+          'Add uninstalled games to Steam automatically'
```

Comment:
this feature seems to add `all games` to steam, even installed ones that were not added before